javascript/javascript1/week2/weatherWear.js

```diff
@@ -0,0 +1,16 @@
+function weatherClothes(temp){
+    
+    if (temp<0){
+        return "Be ready for expected snow and wear heavy jackets and snow shoes";
+    }
+    else if (temp > 0 && temp <= 10){
```

Comment:
The current implementation is not correct if I call this function with _0_
```suggestion
    else if (temp >= 0 && temp <= 10){
```

Reply:
oh i see will change it right away